poketechnika/Game.cpp

```diff
@@ -59,27 +59,28 @@ void Game::processInput() {
     }
 }
 
-void Game::update() {
-    for (int i = 0; i < gameObjects.size(); i++)
-    {
-        gameObjects[i]->update();
-    }
-}
+//void Game::update() {
+//    for (int i = 0; i < gom.gameObjectsNumber(); i++)
+//    {
+//        gameObjects[i]->update();
+//    }
+//}
 
 void Game::initGameLoop() {
     //Create all necessary instances before game starts
-    GameObject::setGameObjectsPtr(&gameObjects);
-    GameScene::setGMPtr(&gm);
+    //GameObject::setGameObjectsPtr(&gameObjects);
+    //GameScene::setGMPtr(&gm);
```

Comment:
without this GameScene is not able to get access to map and is throwing an error (null ptr)